src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/InvitationRepository.cs

```diff
@@ -45,5 +45,8 @@ public void RemoveMcPkg(McPkg mcPkg)
 
         public void RemoveParticipant(Participant participant)
             => _context.Participants.Remove(participant);
+
+        public void RemoveAttachment(Attachment attachment)
```

Comment:
Missing test for RemoveAttachment in InvitationRepositoryTests.
When investigating InvitationRepositoryTests, I see a lot of test missing there + a lot of existing tests not belonging there.

Reply:
... call me if you don't see test not belonging there

Reply:
I have a test, RemoveAttachment_KnownAttachment_ShouldRemoveAttachment, in InvitationRepositoryTests. Should I add more?